tools/topology/topology2/platform/intel/hw_config_cardinal_clk.conf

```diff
@@ -1,15 +1,16 @@
-# Hardware config class definition DAIs. All attributes defined herein
+# Hardware config class definition. All attributes defined herein
 # are namespaced by alsatplg to "Object.Base.hw_config.NAME.attribute_name"
 #
-# Object.Base.hw_config."NAME" {
+# Object.Base.hw_config.1 {
 # 		id		0
-# 		mclk_freq	24000000
-# 		bclk_freq	4800000
-# 		tdm_slot_width	25
+# 		bclk_freq	3072000
+#		tdm_slot_width  32
 # 	}
 #
-# where NAME is the unique instance name for the hw_config object
-# within the same alsaconf node.
+# where 1 is the unique instance ID for the hw_config object within the same alsaconf
```

Comment:
Why use a number when a name is more descriptive ?

Reply:
@lgirdwood all objects need to have instance so that we can use arrays.